deb-get

```diff
@@ -2505,14 +2505,14 @@ else
 fi
 
 case "${UPSTREAM_CODENAME}" in
```

Comment:
Would be better to just test for a lowercased codename using `${UPSTREAM_CODENAME,,}` rather the regex everything.

Reply:
Agreed. Closing PR.